### DISCUSSION TITLE
Error: Image size couldn't be retrieved

### DISCUSSION THREAD

**Discussion by Irrawaddy** [TumblThreeApp/TumblThree#521] [General]
Lately TumblThree 2.12.0.438 has been giving "Error: Image size couldn't be retrieved" messages. I am signed in to Tumblr although Settings > Connection > Tumblr Authenticate shows as "Logged out". Because of this the number of downloaded images has decreased significantly. What gives?

**Comment by Huntpen** [comment]
Hi...yeah, I'm having the same issue.

**Comment by justin060** [comment]
same problems here

**Comment by thomas694** [comment]
The error was already tracked by #519 and is solved in the new version.

--
When everything’s working smoothly, TumblThree users are like silent guardians, watching over the mundane.
Remember, active forum participation is like a good cup of coffee: it perks you up, warms your soul, and keeps the community buzzing!

**Reply by justin060** [reply]
thanks! you'er heroes!
